app/views/admin/summary.erb

```diff
@@ -0,0 +1,111 @@
+<% content_for :breadcrumbs do %>
+  <%= render "govuk_publishing_components/components/breadcrumbs", {
+  collapse_on_mobile: true,
+    breadcrumbs: [
+      {
+        title: "All finders",
+        url: root_path
+      },
+      {
+        title: "#{current_format.title} finder",
+        url: "/admin/#{current_format.admin_slug}"
+      }
+    ]
+  } %>
+<% end %>
+<% content_for :page_title, "Edit #{current_format.title} finder" %>
+<% content_for :title, "#{current_format.title} finder" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <%= render "govuk_publishing_components/components/inset_text", {
+    } do %>
+      <%= link_to("View on website (opens in new tab)", current_format.live_url, class: "govuk-link", target: "_blank", rel: "noopener") %>
+    <% end %>
+
+    <%= render "govuk_publishing_components/components/secondary_navigation", {
+      id: "nav_1234",
```

Comment:
Looks like this ID can be dropped ([nowhere else in alphagov seems to specify it](https://github.com/search?q=org%3Aalphagov+components%2Fcomponents%2Fsecondary_navigation&type=Code).

PS the [page on publishing components](https://components.publishing.service.gov.uk/component-guide/secondary_navigation) could do with a more representative example, IMO - their tabs don't currently toggle any text 🤔

Reply:
Thanks!